docs/maintainer.md

```diff
@@ -17,7 +32,7 @@ We define *stale* as issues that are 30 days or older. Stale is often an indicat
 * *broad* - create more focused issues, such as "add flow types to `WhyPaused` component"
 * *low priority* - close it and add it to the [enhancements board]
 
-#### Making Issues `available`
+#### Making features `available`
```

Comment:
features?

Reply:
i wanted to differentiate btw bugs and features... but it might be a bad dstinction...

Reply:
yeah ...  i think of bugs/ features as all issues, we probably need to make the distinction clearer in the doc . If we are changing it here, might need to re-word the section below it as well.

Reply:
Why restrict the process to features (enhancements) and not also to bugs? Bug reports can also go stale, not just feature request. So I'd stick to "Issues" here instead of only "feature requests".